pkg/network/state.go

```diff
@@ -228,6 +237,9 @@ type client struct {
 	lastFetch time.Time
 	closed    *closedConnections
 	stats     map[StatCookie]StatCounters
+
+	// Flag for handling closed connection capacity pressure for this client
+	closedConnectionsNearCapacity bool
```

Comment:
Do we need to keep this around? Why not just calculate this every time we need to?

Reply:
removed